content/blog/2020/05/citrix-policy-migration.md

```diff
@@ -0,0 +1,167 @@
++++
+author = "Scott Caster	"
+author_url = "https://github.com/scaster513"
+categories = ["Scott Caster", "Citrix", "Policy", "Migrate"]
+date = "2020-05-20"
+description = "Simple process for policy migration from xenapp 6.5 to Citrix 7.xx"
+featured = ""
+featuredalt = ""
+featuredpath = ""
+linktitle = ""
+title = "Migrate Citrix Policies from XenApp 6.5 to >7.6"
+type = "post"
+
++++
+
+## Migrate Citrix Policies from XenApp 6.5 to >7.6
+In this post I will outline the process I used to overcome the limitations of existing Citrix policy migrating tools. Per Citrix, there is no direct path to migrate policies from xenapp version 6.5 to 7.7 or above, you must first install and migrate to 7.6, then do an in-place upgrade to the desired version. (source: [docs.citrix.com](https://docs.citrix.com/en-us/xenapp-and-xendesktop/7-15-ltsr/upgrade-migrate/xenapp-worker-upgrade.html)). Unfortunately, many discover this only after their brand new 7.7 or higher farm is stood up. If you are in a similar situation or if you would like to try a more streamlined approach, the procedure below will help you save some time. It is quick and relatively painless, however, you will need to use and modify Powershell modules. If you are uncomfortable with PowerShell this is a great learning opportunity. The process is broken down into 4 parts below. 
+
+# Export
+1.	Download the Citrix Migration Tool from the xenapp 7.6 download page: [Citrix 7.6 Downloads](https://www.citrix.com/downloads/xenapp/product-software/xenapp-76-enterprise-edition.html)
+     * Important note, be sure to download the migration "scripts", not the migration "tool". The files we need are ReadIMA.zip and ImportFMA.zip. If you see XDExport.zip and XDImport.zip, you have the the actual tool and it will not work. 
```

Comment:
```suggestion
     * Important note, be sure to download the migration "scripts", not the migration "tool". The files we need are `ReadIMA.zip` and `ImportFMA.zip`. If you see `XDExport.zip` and `XDImport.zip`, you have the the actual tool and it will not work. 
```